caravel/static/default.css

```diff
@@ -89,3 +95,38 @@ form .thumbnail {
 .claim-listing-button {
     display: inline-block;
 }
+
+/* Listings table */
+.listings-table {
+    margin-bottom: 0px;
+}
+
+.listing-table-image {
+    width: 104px;
+}
+
+
+/* List-Thumbail togles */
+
+.btn-thumbnail-List-view {
```

Comment:
This feels a little logic-y. How about an `{% if %}` that changes between `.btn-default` and `.btn-primary`?